### PR TITLE
FHAC-634:Update DBScripts and code to comply with db updates for auditrepo

### PR DESCRIPTION
**Pull request by alameluchidambaram**
1. Fix nhincdb mysql scripts. Fix oracle nhincdb scripts
2. Fix Java code to comply with db updates.

### CODE REVIEW

**Comment by TabassumJafri** [comment]
Reviewed the PR .. no comments and ok with merge.

**Comment by alameluchidambaram** [comment]
@TabassumJafri and @jsmith2 
Please Review this PR

File: Product/DBScripts/nhincdb/nhincdb_oracle.sql [hunk 1]
```diff
@@ -64,17 +64,17 @@
...
-	"RECEIVERPATIENTID" VARCHAR2(128 BYTE), 
-	"SENDERPATIENTID" VARCHAR2(128 BYTE), 
-	"COMMUNITYID" VARCHAR2(255 BYTE), 
-	"MESSAGETYPE" VARCHAR2(100 BYTE), 
-	"MESSAGE" BLOB
+	"EVENTTIMESTAMP" DATE NOT NULL, 
+	"EVENTID" VARCHAR2(100 BYTE) NOT NULL, 
+	"USERID" VARCHAR2(100 BYTE) DEFAULT NULL, 
+	"EVENTTYPE" VARCHAR2(100 BYTE) NOT NULL,
+    "OUTCOME" NUMBER(3,0) NOT NULL,
```
Comment (jsmith2) [inline Product/DBScripts/nhincdb/nhincdb_oracle.sql]: Formatting seems a little off here.

File: Product/Production/Services/AuditRepositoryCore/src/test/java/gov/hhs/fha/nhinc/auditrepository/hibernate/AuditRepositoryDAOTest.java [hunk 2]
```diff
@@ -85,8 +77,8 @@ public void testInsertAuditRepository() {
         Date now = new Date();
 
         AuditRepositoryRecord record = new AuditRepositoryRecord();
-        record.setMessageType("Record 1 - Message Type");
-        record.setTimeStamp(now);
+        record.setDirection("Record 1 - Message Type");
```
Comment (jsmith2) [inline Product/Production/Services/AuditRepositoryCore/src/test/java/gov/hhs/fha/nhinc/auditrepository/hibernate/AuditRepositoryDAOTest.java:83]: I realize that we have another ticket for updating the adapter when the schema is updated.  But feel we should at least add a TODO in the unit test to make sure we don't forget to update the tests later when it's changed.

File: Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/auditrepository/nhinc/AuditRepositoryOrchImpl.java [hunk 3]
```diff
@@ -95,72 +94,39 @@ public AuditRepositoryOrchImpl() {
...
-                auditRec.setParticipationTypeCodeRole(eventParticipationTypeCodeRole);
-                eventParticipationIDTypeCode = participantObjectIdentificationType.getParticipantObjectIDTypeCode()
-                    .getCode();
-                auditRec.setParticipationIDTypeCode(eventParticipationIDTypeCode);
-            }
+            auditRec.setRemoteHcid("");
         }
 
-        auditRec.setMessageType(mess.getInterface() + " " + mess.getDirection());
+        auditRec.setDirection(mess.getInterface() + " " + mess.getDirection());
```
Comment (jsmith2) [inline Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/auditrepository/nhinc/AuditRepositoryOrchImpl.java:118]: won't this setDirection be overridden by the one below?  auditRec.setDirection("EventId");

File: Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/auditrepository/nhinc/AuditRepositoryOrchImpl.java [hunk 4]
```diff
@@ -95,72 +94,39 @@ public AuditRepositoryOrchImpl() {
...
         XMLGregorianCalendar xMLCalDate = eventIdentification.getEventDateTime();
         if (xMLCalDate != null) {
             eventTimeStamp = convertXMLGregorianCalendarToDate(xMLCalDate);
-            auditRec.setTimeStamp(eventTimeStamp);
+            auditRec.setEventTimeStamp(eventTimeStamp);
         }
+        auditRec.setDirection("EventId");
+        auditRec.setEventType("EventType");
+        auditRec.setOutcome(0);
+        auditRec.setDirection("Nhin Outbound");
```
Comment (jsmith2) [inline Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/auditrepository/nhinc/AuditRepositoryOrchImpl.java]: Actually you have setDirection 3 times.

**Comment by jsmith2** [comment]
Merging with no further comments.

**Pull request merged**